src/shared_modules/utils/windowsHelper.h

```diff
@@ -175,6 +192,52 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(const std::string& timestamp1, const std::string& timestamp2)
+    {
+        std::string normalizedTimestamp;
+
+        if (timestamp1.size() == INSTALLDATE_REGISTRY_VALUE_SIZE)
```

Comment:
if timestamp1 have an invalid format ? non-number, size != INSTALLDATE_REGISTRY_VALUE_SIZE ?

Reply:
It will return an empty string. I could set the value to the timestamp retrieved by the function keyModificationDate() if you prefer that approach.